### PR TITLE
Todolist 3 setup webpack to work with js s css html

### PR DESCRIPTION
**Pull request by Kanagen**
Я, в принципе разобрался с основными настройками webpack. И документация у него хорошая. Все понятно.
Написал README. Буду изучать следующие задания.

### CODE REVIEW

File: src/Style/style.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+}
+body {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.container {
```
Comment (vovank86) [inline src/Style/style.css:14]: Remove all begining from this line

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <!-- контейнер для календаря -->
```
Comment (vovank86) [inline src/index.html:10]: Remove all from body. For now it should be empty.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
+import './Style/style.css'
```
Comment (vovank86) [inline src/index.js:1]: Please change folder name './Style/style.css' -> './styles/style.css'

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
```
Comment (vovank86) [inline src/index.js:2]: Remove all beginning from this line

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        static: './dist',
+    },
+    module:{
+        rules:[
+            {
+                test: /\.css$/i,
+                use: [MiniCssExtractPlugin.loader, "css-loader"],
+              },
+            // {
+            //     test:/\.css$/i,
```
Comment (vovank86) [inline webpack.config.js:23]: Remove comments

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                test:/\.html&/i, 
+                use: 'html-loader'
+            }
+        ]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html',
+            title: 'Calendar',
```
Comment (vovank86) [inline webpack.config.js:43]: Please change title to "TO-DO List"

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+            filename:'calendar.html'
```
Comment (vovank86) [inline webpack.config.js:44]: Please remove filename. The entry point for the site/application is always index.html

File: webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        clean: true
+    },
+    devServer:{
+        static: './dist',
+    },
+    module:{
+        rules:[
+            {
+                test: /\.css$/i,
+                use: [MiniCssExtractPlugin.loader, "css-loader"],
```
Comment (vovank86) [inline webpack.config.js:20]: If you want to try module css this rule should be modified. (It's optional)

File: package.json [hunk 9]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
+  "scripts": {
```
Comment (vovank86) [inline package.json:6]: "watch": "webpack --watch",
  "serve": "webpack serve --open",
   "build": "webpack",
   "start": "npm run serve & npm run watch"

File: README.md [hunk 10]
```diff
@@ -1 +1,5 @@
-# todo-list
```
Comment (vovank86) [inline README.md:1]: Please describe this only steps to start development. 
e.g. 
# TO-DO List

---
To star development first of all install all needed dependences. To do it open the terminal and run the command ```npm install``` 

To start the dev-server please run next command ```npm run start```

File: src/index.html (hunk 2 above)
Comment (vovank86) [reply src/index.html:10]: You can add  <h1> TODO  List</h1>

File: webpack.config.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+              },
+            {
+                test:/\.html&/i, 
+                use: 'html-loader'
+            }
+        ]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html',
```
Comment (vovank86) [inline webpack.config.js:42]: Please use path generator for template such as: `path.resolve(__dirname, "src/index.html")`

File: webpack.config.js [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+            title: 'ToDoList',
+            filename:'index_bandle.html'
```
Comment (vovank86) [inline webpack.config.js:44]: Please remove filename property at all

File: package.json (hunk 9 above)
Comment (vovank86) [reply package.json:6]: Please update scripts as in prev comment

File: src/index.js (hunk 3 above)
Comment (Kanagen) [reply src/index.js]: Done

File: src/index.js (hunk 4 above)
Comment (Kanagen) [reply src/index.js]: Done

File: webpack.config.js (hunk 5 above)
Comment (Kanagen) [reply webpack.config.js:23]: Done

File: webpack.config.js (hunk 6 above)
Comment (Kanagen) [reply webpack.config.js]: Done

File: webpack.config.js (hunk 7 above)
Comment (Kanagen) [reply webpack.config.js]: Done

File: webpack.config.js (hunk 8 above)
Comment (Kanagen) [reply webpack.config.js:20]: Please remind where I can fide it in webpack docs

File: package.json (hunk 9 above)
Comment (Kanagen) [reply package.json:6]: Done

File: README.md (hunk 10 above)
Comment (Kanagen) [reply README.md:1]: Done

File: webpack.config.js (hunk 11 above)
Comment (Kanagen) [reply webpack.config.js:42]: Done

File: webpack.config.js (hunk 12 above)
Comment (Kanagen) [reply webpack.config.js:44]: Done

**Pull request merged**